### PR TITLE
Further minor updates for Data Release 10

### PR DESCRIPTION
**Pull request by geordie666**
This PR supplements #165 by making some small modifications to the DR10 description page in response to comments from @djschlegel.

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: My understanding is that once we've merged this PR we can serve the DR10 pages.

File: CHANGES.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 ## 10.0.0 (DR10, 2022-12-20)
 
 - Near-final updates for DR10
-  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)):
+  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)) and
+  ([PR#166](https://github.com/legacysurvey/legacysurvey/pull/166)):
```
Comment (weaverba137) [inline CHANGES.md:14]: Since we've already tagged 10.0.0, this PR will go into 10.0.1.

**Review by weaverba137 — APPROVED** [review]
One minor comment on the change log.  Please fix and merge when ready.

File: CHANGES.md (hunk 1 above)
Comment (geordie666) [reply CHANGES.md]: Good point - fixed.

**Pull request merged**